src/build.ts

```diff
@@ -0,0 +1,15 @@
+import { optimize } from "./optimize"
+import { createIndexFiles } from "./createIndexFiles"
+import { components } from "./components"
+import { clean } from "./clean"
+
+import { AssetsConfig } from "./index"
+
+export const build = (args, config: AssetsConfig) => {
+  clean(args, config)
```

Comment:
I would expect the build to be incremental and not remove everything. Worth documenting.